roles/common/tasks/file_tests.yml

```diff
@@ -1,10 +1,12 @@
 ---
-- name: Get stats for file - "{{ item }}"
+- name: |
+    TEST Check stats for file -- "{{ item }}"
+    {{ common_file_test_id }}
   ansible.builtin.stat:
     path: "{{ item }}"
   register: fstats
-  failed_when: 
-    - fstats.stat.pw_name != "root"
-    - fstats.stat.size | int < 300
-    - not fstats.stat.exists
-    - not fstats.stat.isreg
\ No newline at end of file
+  failed_when:
+    - ( (not fstats.stat.exists) or
+      (fstats.stat.pw_name != "root") or
+      (fstats.stat.size | int < 300) or
+      (not fstats.stat.isreg) )
```

Comment:
This is covered in https://github.com/infrawatch/feature-verification-tests/pull/204/, which is pending some updates.

Reply:
Technically, this change can be tackled separately, and the fix falls outside the scope of this PR.

I will remove this part of the file test